cmd/multinode/main.go

```diff
@@ -93,22 +92,15 @@ func main() {
 func init() {
 	defaultConfDir := fpath.ApplicationDir("storj", "multinode")
 	cfgstruct.SetupFlag(zap.L(), rootCmd, &confDir, "config-dir", defaultConfDir, "main directory for multinode configuration")
-	cfgstruct.SetupFlag(zap.L(), rootCmd, &identityDir, "identity-dir", "", "main directory for multinode identity credentials")
```

Comment:
I would suggest we still maintain this flag for backward compatibility reasons. We've already marked it deprecated and hidden.